src/kak-setsid.c

```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
+#include <unistd.h>
+
+void execute_command_in_arguments(int argc, char *argv[]);
+
+int main(int argc, char *argv[])
+{
+    if(argc < 2) {
+        fprintf(stderr, "Give a command to execute as argument to %s.\n",
+                argv[0]);
+        return 1;
+    }
+    
+    switch(fork()) {
+        case -1:
+            fprintf(stderr, "Failed to create child process.\n");
```

Comment:
Use `perror("kak-setsid: fork");` here, as that will explain why fork failed.

Reply:
I didn't know about `perror()`. Thanks for help!